SVG and raster images can be embedded in project (#4563)
qgis/QGIS-Documentation: source/docs/user_manual/introduction/qgis_configuration.rst

System Settings
---------------

.. _svg_paths:

**SVG paths**

Add or Remove :guilabel:`Path(s) to search for Scalable Vector Graphic (SVG)
symbols`. These SVG files are then available to symbolize or label the features
or decorate your map composition.

When using an SVG file in a symbol or a label, QGIS allows you to:

> * load the file from the disk: the file is identified through the file path and

Reviewer: ```suggestion
* load the file from the file system: the file is identified through the file path and
```

  QGIS needs to resolve the path in order to display the corresponding image

> * load the file from a remote URL: as above, on the success of the connection

Reviewer 2: ```suggestion
* load the file from a remote URL: as above, the image will only be loaded on successful retrieval of the remote resource
```

>   depends the image retrieval

Reviewer: ```suggestion
```

> * embed the SVG file into the item: the file is saved as a base64 string within

Reviewer 2: "the file is saved as a base64 string within"

Is this relevant for users? It's not exposed anywhere, and could potentially be changed in future.

Reviewer 2: Maybe just "the file is embedded inside the current project, style database, or print layout template."

> * embed the SVG file into the item: the file is saved as a base64 string within

Reviewer: ```suggestion
* embed the SVG file into the item: the file is embedded inside
```

>   the symbol or label, and also in the project, an associated style or a print

Reviewer: ```suggestion
  the current project, style database, or print layout template.
```

>   layout template file. THe SVG file is then always rendered as part of the item.

Reviewer 2: ```suggestion
  layout template file. The SVG file is then always rendered as part of the item.
```

>   layout template file. THe SVG file is then always rendered as part of the item.

Reviewer: ```suggestion
```

Author: @havatv you'd remove the second sentence?

>   A convenient way to share projects with custom SVG symbols.

Reviewer 2: ```suggeston
  This is a convenient way to create self-contained projects with custom SVG symbols which can be easily shared amongst different users and installations of QGIS.
```

>   A convenient way to share projects with custom SVG symbols.

Reviewer: ```suggestion
  This is a convenient way to create self-contained projects with custom SVG symbols
  which can be easily shared amongst different users and installations of QGIS.
```

>   It's as well possible to extract the embedded SVG file from a symbol or label

Reviewer 2: ```suggestion
  It is also possible to extract the embedded SVG file from a symbol or label
```

  and save it on disk.

> .. note:: The abovementioned ways of loading/storing an SVG file in a project

Reviewer 2: ```suggestion
.. note:: The above mentioned options for embedding an SVG file in a project
```

Author: It's not only the embedding option I'm referring to: the options to load from file disk and URL are intended to be included, too.
